Gemfile

```diff
@@ -21,3 +21,5 @@ gem "yard", "~> 0.9", require: false
 
 # The Rails documentation link only activates when railties is detected.
 gem "railties", "~> 7.0", require: false
+
+gem "rspec", "~> 3.11", require: false
```

Comment:
`rspec` is added because Rubocop tests are written using RSpec. I'm open to discussing about this. As far as I'm aware, Rubocop doesn't provide test helpers in minitest. I'm not sure how much effort it would take to write the Rubocop tests from scratch in minitest.

If this is acceptable, it needs to be added to CI.

Reply:
https://github.com/rubocop/rubocop-minitest doesn't use RSpec for its tests - you could take a look how that's done.

Reply:
Yeah, let's please avoid adding rspec as a dependency.

Reply:
We unload RuboCop to test that Syntax Tree formatting also works. Requiring rubocop in setup will make sure it passes regardless of the order in which they run.

```ruby
def setup
  require "rubocop"
end
```